web/tools/webpack.config.ts

```diff
@@ -285,7 +285,7 @@ const createConfig = (
   }
 
   // Optimize the bundle in production mode
-  if (!devServer && optimize) {
```

Comment:
This changes the meaning of the code I suppose.

Reply:
optimize is just a namespace/import from webpack, I don't really see a reason why we need to check for it.